src/components/cluster/ClusterInfoStepsModal.tsx

```diff
@@ -0,0 +1,97 @@
+import Tippy from '@tippyjs/react'
+import { copyToClipboard } from '../common'
+import 'tippy.js/themes/light.css'
+import { ReactComponent as Help } from '../../assets/icons/ic-help.svg'
+import './cluster.scss'
+import { ReactComponent as ClipboardIcon } from '../../assets/icons/ic-copy.svg'
+import { ClusterStepModal } from './cluster.type'
+import { DOCUMENTATION } from '../../config'
+
+function ClusterInfoSteps({ command, clusterName }: ClusterStepModal) {
+    const [copied, setCopied] = useState(false)
+    const copyClipboard = (e): void => {
+        e.stopPropagation()
+        setCopied(true)
+        copyToClipboard(command)
+    }
+    const infoItems = [
+        {
+            additionalInfo: (
+                <div>
+                    <span className="fw-6">Prerequisites:</span> kubectl and Jq should be installed
+                </div>
+            ),
+        },
+        {
+            info: 'Run below command on terminal to get server URI & bearer token',
+            additionalInfo: (
+                <div className="dc__position-rel cluster-modal-container dc__align-left bcn-1 p-10 lh-20 mt-2 br-4">
+                    <div className="fs-13 fw-4 h-100 dc__overflow-scroll" id="command-code">
+                        {command}
+                    </div>
+                    <Tippy
+                        className="default-tt p-4"
+                        arrow={false}
+                        placement="bottom"
+                        content={copied ? 'Copied!' : 'Copy'}
+                        trigger="mouseenter click"
+                        onShow={(_tippy) => {
+                            setTimeout(() => {
+                                _tippy.hide()
+                                setCopied(false)
+                            }, 5000)
+                        }}
+                        interactive={true}
+                    >
+                        <div className="cluster-clipboard dc__position-abs cursor" onClick={copyClipboard}>
+                            <ClipboardIcon className="icon-dim-20" />
+                        </div>
+                    </Tippy>
+                </div>
+            ),
+        },
+        {
+            info: 'Copy & paste Server URL & Bearer token from command output',
+        },
+        {
+            info: `Replace Local IP with public IP at which ${clusterName} cluster api server is accessible`,
+        },
+    ]
+
+    return (
+        <div className="p-12 fs-13 fw-4">
+            {infoItems.map((item, key) => (
+                <div className="cluster-modal-wrapper ">
+                    <div className="cluster-modal-number flex mr-16 bw-1 bcn-0 en-2 icon-dim-24">{key + 1}</div>
+                    <div
+                        className={`cluster-inner-container flexbox-col dc__border-left pt-2 pr-44 pb-20 pl-28 lh-20 dc__align-start dc__content-start ${
+                            infoItems.length - 1 === key ? 'cluster-last-item' : ''
```

Comment:
can we leverage last child pseudo selector instead of doing this?

Reply:
Done